client/components/Item/Card/Card.vue

```diff
@@ -202,6 +206,35 @@ export default Vue.extend({
 
       return '';
     },
+    /**
+     * @returns {string} A link to be applied to the title
+     */
```

Comment:
Do we want full sentences for the docs or just capital letters at the start? Honestly not sure what the plan is for this project specifically.

Reply:
@mrtimscampi

Reply:
We seem to be using capital letters almost everywhere, so that seems to be the style we're going for.

For sentences or not, it seems to vary a bit. I'm leaning towards just describing what it returns, instead of what it should be used for. So here, something like `A router link to the item's page` or something similar, maybe?

Reply:
Though it's not always the item's page ^^

Reply:
True, but you get the idea :yum:  Basically describe the thing it returns instead of how it's used :smile: